app/views/returnFrequency/ChangeReturnFrequencyConfirmationView.scala.html

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  *@
 
-@import views.html.templates.updateClient
+@import views.html.templates.UpdateClient
```

Comment:
Could you remove `View` from the end of the filename to match the rest of the views please?